src/Couchbase.Lite.Shared/API/Error/CouchbaseLiteErrorMessage.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Couchbase.Lite
+{
+    internal static class CouchbaseLiteErrorMessage
+    {
+        //Database error messages
+        internal const string CreateDBDirectoryFailed = "Unable to create database directory";
+        //Database - Copy
+        internal const string ResolveDefaultDirectoryFailed = "Failed to resolve a default directory! If you have overriden the IDefaultDirectoryResolver interface, please check it.  Otherwise please file a bug report.";
```

Comment:
```suggestion
        internal const string ResolveDefaultDirectoryFailed = "Failed to resolve a default directory! If you have overriden the default directory, please check it.  Otherwise please file a bug report.";
```